A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/systems.cfg

```diff
@@ -45,6 +45,7 @@ lightdef.38 = Type:12#Index:1#LocalPosition:38.5,-1.825,5.980#LocalRotation:104,
 lightdef.39 = Type:12#Index:2#LocalPosition:38.5,1.760,5.980#LocalRotation:104,0,90#EffectFile:LIGHT_A32NX_ScreenOrange#Node:#PotentiometerIndex:11
 lightdef.40 = Type:13#Index:2#LocalPosition:35.0,0.960,9.000#LocalRotation:110,90,0#EffectFile:LIGHT_A32NX_CockpitSpotNarrow#PotentiometerIndex:97
 lightdef.41=  Type:13#Index:2#LocalPosition:34.488,-1.160,9.040#LocalRotation:110,90,0#EffectFile:LIGHT_A32NX_CockpitSpotNarrow#PotentiometerIndex:96
+lightdef.42 = Type:13#Index:0#LocalPosition:36.5,0,6#LocalRotation:0,0,0#EffectFile:LIGHT_A32NX_CockpitMinimalAmbiantLarge#PotentiometerIndex:1
```

Comment:
Probably I will fix all the issues with my PR today so either you will merge and change the values or I will merge and you will change it. Let me know your thoughts.

Reply:
> Probably I will fix all the issues with my PR today so either you will merge and change the values or I will merge and you will change it. Let me know your thoughts.

Yeah exactly just a small change of light number should clear any conflicts, since I am using a newly imported effect, it should not conflict any further.